app/controllers/tasks_controller.rb

```diff
@@ -1,16 +1,24 @@
 class TasksController < ApplicationController
 
+    rescue_from ::ActiveRecord::RecordNotFound, with: :record_not_found
+
     def create
         task = Task.new(task_params)
         if task.save
-            render json: task
+            render_response(task)
         else
-            render json: {error: "The task couldn't be added. Try again!"}
+            render_response("The task couldn't be added. Try again!")
```

Comment:
Great! Again.. doesn't really make the code shorter..BUT you made a reusable method 🎉